package.json

```diff
@@ -25,6 +25,7 @@
     "electron-rebuild": "electron-rebuild",
     "flow": "flow; test $? -eq 0 -o $? -eq 2",
     "lint": "standard --verbose | snazzy",
+    "lint-standard": "standard",
```

Comment:
why do we need another standard linter command?

Reply:
because the `lint-stanadrd` command generates output so that it can be parsed by programs that will position the editor to each error. when you do the `--verbose | snazzy` thing, you lose that.

Reply:
ok for me it's working with normal lint as well

Reply:
you probably don't use emacs

Reply:
no, using vsc 😃